libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2344,7 +2344,7 @@ bool AP_AHRS::attitudes_consistent(char *failure_msg, const uint8_t failure_msg_
             primary_quat.angular_difference(dcm_quat).to_axis_angle(angle_diff);
             const float yaw_diff = fabsf(angle_diff.z);
             if (check_yaw && (yaw_diff > ATTITUDE_CHECK_THRESH_YAW_RAD)) {
-                hal.util->snprintf(failure_msg, failure_msg_len, "DCM Yaw differs from current AHRS by %d deg", (int)degrees(yaw_diff));
+                hal.util->snprintf(failure_msg, failure_msg_len, "DCM: Yaw inconsistent, Wait or check sensors");
```

Comment:
DCM: Yaw difference. Wait or check sensors.

Reply:
and again